corehq/apps/userreports/management/commands/prune_old_datasources.py

```diff
@@ -69,7 +69,7 @@ def handle(self, **options):
                         except Exception:
                             print("\t{}: no inserted_at column, probably not UCR".format(tablename))
                         else:
-                            print("\t{}: {}".foramt(tablename, result.fetchone()))
+                            print("\t{}: {}".format(tablename, result.fetchone()))
```

Comment:
This morning I was going to facetiously suggest that we install [this library](https://github.com/asottile/future-fstrings) just to help us make it through the next 21 days. :upside_down_face:  No more `format()`.